### PR TITLE
Bugfix/lattice 2115 id constants entity set

### PR DESCRIPTION
**Pull request by you-leee**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/assembler/AssemblerQueryService.kt [hunk 1]
```diff
@@ -47,7 +47,7 @@ class AssemblerQueryService(private val edmService: EdmManager) {
                           srcAggregates: Map<String, List<AggregationType>>, edgeAggregates: Map<String, List<AggregationType>>, dstAggregates: Map<String, List<AggregationType>>,
                           calculations: Set<Calculation>,
                           srcFilters: Map<String, List<Filter>>, edgeFilters: Map<String, List<Filter>>, dstFilters: Map<String, List<Filter>>
-    ): Iterable<Map<String, Any>> {
+    ): Iterable<Map<String, Any?>> {
```
Comment (katherinebernstein) [inline src/main/kotlin/com/openlattice/assembler/AssemblerQueryService.kt]: Does this ever return null?

Comment (you-leee) [reply src/main/kotlin/com/openlattice/assembler/AssemblerQueryService.kt]: Grouping (and selected) values can be null, if I remember right. There is a section in the mapper of the PostgresIterable returned where if the grouping column has no value, the getArray will return null

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/assembler/AssemblerQueryService.kt]: okay -- I think this will require datastore updates then

Comment (you-leee) [reply src/main/kotlin/com/openlattice/assembler/AssemblerQueryService.kt]: Yeah, you'r right. I will make a branch there to to get it compile

**Pull request merged**